app/controllers/users_controller.rb

```diff
@@ -16,6 +16,8 @@ def create
 
   def show
     @user = User.find(session[:current_user_id])
+    @post = Post.new
+    @posts = Post.all
```

Comment:
@posts = Post.new.all?????

Reply:
The view users/show.html.erb needs these 3 variables set up in the controller so they can be used. @post is needed for the post#create route in line 8 of that view.